Update end-user-access-cloud-pc.md (#4211)
MicrosoftDocs/memdocs: windows-365/end-user-access-cloud-pc.md

To set up their Remote Desktop client, users follow these steps:

1. Download the Remote Desktop app from the [Remote Desktop clients page](/windows-server/remote/remote-desktop-services/clients/remote-desktop-clients).

> 2. Select **Subscribe**. (If use macOS, iOS, iPadOS, Android/Chrome OS Select **Add Workspace** and type thease URL `https://rdweb.wvd.azure.us/api/arm/feeddiscovery`)

Reviewer: ```suggestion
2. Select **Subscribe**. (If using macOS, iOS, iPadOS, or Android/Chrome OS, select **Add Workspace** and then enter the URL `https://rdweb.wvd.azure.us/api/arm/feeddiscovery`).
```
